frontend/rolecall/src/app/cast/cast-editor-v2.component.html

```diff
@@ -1,34 +1,48 @@
 <app-loading-spinner *ngIf="!dataLoaded"></app-loading-spinner>
-<div [ngStyle]="dataLoaded ? {} : { 'display' : 'none' }" class="root-container">
+<div class="root-container"
+    [ngStyle]="dataLoaded ? {} : { 'display' : 'none' }">
   <div class="cast-selector-container">
     <div class="app-card card-margins left-card">
-      <div ngbDropdown #pieceDropdown="ngbDropdown">
-        <div class="app-card piece-select" ngbDropdownToggle>
-          <h1 class="left-card-title">
+      <div ngbDropdown
+          #pieceDropdown="ngbDropdown">
+        <div ngbDropdownToggle
+            class="app-card piece-select">
+          <h1 class="left-card-title"
+              [ngStyle]="{'font-style': selectedPiece && selectedPiece.siblingId > 0 ? 'italic' : 'normal' }">
             {{selectedPiece ? selectedPiece.name : " "}}
           </h1>
           <mat-icon class="dropdownArrow">
             arrow_drop_down
           </mat-icon>
-          <div ngbDropdownMenu aria-labelledby="Piece Dropdown" class="piece-options">
-            <button ngbDropdownItem *ngFor="let piece of allPieces" (click)="onSelectPiece(piece)">
+          <div ngbDropdownMenu
+              class="piece-options"
+              aria-labelledby="Piece Dropdown">
+            <button *ngFor="let piece of popupList"
+                ngbDropdownItem
+                (click)="onSelectPiece(piece.pieceIx)"
```

Comment:
Can we rename `pieceIx` to `pieceIndex`, shortened words are not great for readability since not everyone has the same background of "norms"

Reply:
I already commented on this below. This would be a biggie. I don't have many conventions, but one is to use Ix for Index everywhere. I doubt anyone can read code of mine without rapidly understanding that Ix means Index. Am I breaking a hard google rule or is this more of a soft guideline?

Reply:
Actually the first time I read your PR with "Ix" my first thought was "intersection", not index, until I got quite a lot deeper into the code and started noticing more patterns which "clicked" for me.

Don't worry about refactoring the current usage if the impact is too large so it doesn't grow the current PR scope, just avoid using it in new places and I can help adjust the existing places separately.

Reply:
OK.